integration/mediation-tests/tests-common/integration-test-utils/src/main/java/org/wso2/esb/integration/common/utils/Utils.java

```diff
@@ -21,6 +21,8 @@
 import org.apache.axiom.om.OMElement;
```

Comment:
Shall we update the license. it should the the year created only.

Reply:
Fixed